tests/users/test_admin.py

```diff
@@ -134,6 +137,86 @@ def test_get_fields_to_transfer_for_job_seekers():
+
+    post_data = {
+        "username": user.username,
+        "email": user.email,
+        "last_login_0": "21/02/2025",
+        "last_login_1": "09:46:54",
+        "date_joined_0": "19/02/2025",
+        "date_joined_1": "18:00:47",
+        "initial-date_joined_0": "19/02/2025",
+        "initial-date_joined_1": "18:00:47",
+        "last_checked_at_0": "19/02/2025",
+        "last_checked_at_1": "18:00:47",
+        "initial-last_checked_at_0": "19/02/2025",
+        "initial-last_checked_at_1": "18:00:47",
+        "title": "",
+        "phone": "",
+        "address_line_1": "",
+        "address_line_2": "",
+        "post_code": "",
+        "department": "",
+        "city": "",
+        "created_by": "",
+        "emailaddress_set-TOTAL_FORMS": "0",
+        "emailaddress_set-INITIAL_FORMS": "0",
+        "emailaddress_set-MIN_NUM_FORMS": "0",
+        "emailaddress_set-MAX_NUM_FORMS": "0",
+        f"{membership_field_name}_set-TOTAL_FORMS": "3",
+        f"{membership_field_name}_set-INITIAL_FORMS": "3",
+        f"{membership_field_name}_set-MIN_NUM_FORMS": "0",
+        f"{membership_field_name}_set-MAX_NUM_FORMS": "0",
+        f"{membership_field_name}_set-0-id": admin_active_membership.pk,
+        f"{membership_field_name}_set-0-user": user.pk,
+        f"{membership_field_name}_set-1-id": active_membership.pk,
+        f"{membership_field_name}_set-1-user": user.pk,
+        f"{membership_field_name}_set-2-id": inactive_membership.pk,
+        f"{membership_field_name}_set-2-user": user.pk,
+        "job_applications_sent-TOTAL_FORMS": "0",
+        "job_applications_sent-INITIAL_FORMS": "0",
+        "job_applications_sent-MIN_NUM_FORMS": "0",
+        "job_applications_sent-MAX_NUM_FORMS": "0",
+        "utils-pksupportremark-content_type-object_id-TOTAL_FORMS": "1",
+        "utils-pksupportremark-content_type-object_id-INITIAL_FORMS": "0",
+        "utils-pksupportremark-content_type-object_id-MIN_NUM_FORMS": "0",
+        "utils-pksupportremark-content_type-object_id-MAX_NUM_FORMS": "1",
+        "utils-pksupportremark-content_type-object_id-0-remark": "",
+        "utils-pksupportremark-content_type-object_id-0-id": "",
+        "utils-pksupportremark-content_type-object_id-__prefix__-remark": "",
+        "utils-pksupportremark-content_type-object_id-__prefix__-id": "",
+        "_save": "Enregistrer",
+    }
+    response = admin_client.post(url, data=post_data)
+    assertRedirects(response, reverse("admin:users_user_changelist"))
+
+    admin_active_membership.refresh_from_db()
+    assert not admin_active_membership.is_active
+    assert not admin_active_membership.is_admin
+    active_membership.refresh_from_db()
+    assert not active_membership.is_active
+    assert not active_membership.is_admin
+    inactive_membership.refresh_from_db()
+    assert not inactive_membership.is_active
+    assert not inactive_membership.is_admin
```

Comment:
Évite le bug assez courant de `assert form.is_valid` (alors qu’il faudrait `assert form.is_valid()`). Pas trop de risque ici, mais bonne pratique :shrug: 
```suggestion
    assert admin_active_membership.is_active is False
    assert admin_active_membership.is_admin is False
    active_membership.refresh_from_db()
    assert active_membership.is_active is False
    assert active_membership.is_admin is False
    inactive_membership.refresh_from_db()
    assert inactive_membership.is_active is False
    assert inactive_membership.is_admin is False
```

Reply:
En effet, merci pour le conseil (je viens d'ailleurs de me faire avoir avec la lib django-otp ou ils ont mis un `user.is_verified()` qui n'est pas une property (contrairement à `user.is_authenticated`)